doc/Command_Index_User.tex

```diff
@@ -1885,25 +1857,301 @@ \subsection{Distances}\label{distances}
+
+
+{\grechangedim{spaceabovelines}{0.2cm}{scalable}
+\grechangedim{spacebeneathtext}{0.2cm}{scalable}
+\begin{center}
+  \scalebox{2}{%
+    \getdims
+    \begin{mplibcode}
+      beginfig(0);
+        pickup pencircle scaled 0.1bp;
+        ahangle := 30; ahlength := 2;
+        y0 := \spacebeneathtext;
+        y1 := y0+\translationheight;
+        y2 := y1+\spacelinestext;
+        y3 := y2+\staffheight;
+        y4 := y3+\abovelinestextraise;
+        y5 := y4+\abovelinestextheight;
+        y6 := y5+\spaceabovelines;
+        x := 0.1cm;
+        draw (-x, 0) -- (x, 0);
+        draw (-x,y0) -- (x,y0);
+        draw (-x,y1) -- (x,y1);
+        draw (-x,y2) -- (x,y2);
+        draw (-x,y3) -- (x,y3);
+        draw (-x,y4) -- (x,y4);
+        draw (-x,y5) -- (x,y5);
+        draw (-x,y6) -- (x,y6);
+        drawdblarrow ( 0, 0) -- (0,y0); label.lft(btex \lab{spacebeneathtext} etex, (0, y0/2));
+        drawdblarrow ( 0,y0) -- (0,y1); label.lft(btex \lab{translationheight} etex, (0, (y0+y1)/2));
+        drawdblarrow ( 0,y1) -- (0,y2); label.lft(btex \lab{spacelinestext} etex, (0, (y1+y2)/2));
+        %drawdblarrow ( 0,y2) -- (0,y3); label.lft(btex \lab{staffheight} etex, (0, (y2+y3)/2));
+        drawdblarrow ( 0,y3) -- (0,y4); label.lft(btex \lab{abovelinestextraise} etex, (0, (y3+y4)/2));
+        drawdblarrow ( 0,y4) -- (0,y5); label.lft(btex \lab{abovelinestextheight} etex, (0, (y4+y5)/2));
+        drawdblarrow ( 0,y5) -- (0,y6); label.lft(btex \lab{spaceabovelines} etex, (0, (y5+y6)/2));
+      endfig;
+    \end{mplibcode}%
+    \hspace{0.2cm}%
+    \lines{\gabcsnippet{<alt>alt</alt>text[trans](g)}}%
+    \hspace{0.5cm}%
+    \lines{\nabcsnippet{text[trans](g|vi)}}%
+  }
+\end{center}
+}
+  
+The height of the staff lines is determined by the commands \verb|\grechangefactor| and \verb|\grechangestafflinethickness|.
+
+\begin{gdimension}{spaceabovelines}
+The space above the staff lines (and above-lines text or NABC neumes, if any).
+\end{gdimension}
+
+\begin{gdimension}{abovelinestextheight}
+Minimum height of the above-lines text. (It must be bigger than the text for it to be taken into consideration.)
+\end{gdimension}
+
+\begin{gdimension}{abovelinestextraise}
+Distance from the top staff line to the baseline of the above-lines text.
+\end{gdimension}
+
+\begin{gdimension}{spacelinestext}
+The space between the lines and the bottom of the text.
+\end{gdimension}
+
+\begin{gdimension}{translationheight}
+The space between the baseline of the text and the baseline of the translation.
+\end{gdimension}
+
+\begin{gdimension}{spacebeneathtext}
+The space beneath the text (and translation, if any).
+\end{gdimension}
+
```

Comment:
When I create the pdf, the entry for `spacebeneathtext` is forced to a new page when there's plenty of room for it on the previous page.  We need to do something to encourage page breaking at this point.

Reply:
That page break was really weird. I inserted a \goodbreak to fix it and hope that's ok.

Reply:
Looks good now.